### PR TITLE
Allow checking for custom handlers outside Dapper library

### PR DESCRIPTION
**Pull request by brendangooden**
Allow checking for whether a type will be handled by a custom handlers outside Dapper library (For use in SimpleCRUD extension)

### CODE REVIEW

**Comment by brendangooden** [comment]
Hi @NickCraver  any updates, can we merge this?

**Comment by NickCraver** [comment]
With the caveat this will break heavily in vNext, no objections to adding it now that we know how severe that break will be. Do you mind filling in the xmldoc here and I'll get it in the next minor release?

File: Dapper/SqlMapper.cs [hunk 1]
```diff
@@ -260,8 +260,12 @@ public static void RemoveTypeMap(Type type)
         /// <param name="type">The type to handle.</param>
         /// <param name="handler">The handler to process the <paramref name="type"/>.</param>
         public static void AddTypeHandler(Type type, ITypeHandler handler) => AddTypeHandlerImpl(type, handler, true);
-
-        internal static bool HasTypeHandler(Type type) => typeHandlers.ContainsKey(type);
+        /// <summary>
+        /// Determine if the specified type will be processed by a custom handler.
+        /// </summary>
+        /// <param name="type"></param>
+        /// <returns></returns>
```
Comment (NickCraver) [inline Dapper/SqlMapper.cs:267]: Needs XML doc loving here please!

**Review by NickCraver — CHANGES_REQUESTED** [review]

**Comment by brendangooden** [comment]
@NickCraver done.

**Review by NickCraver — APPROVED** [review]
:+1: Thanks for the tweaks!

**Pull request merged**